cartographer_ros/cartographer_ros/pbstream_to_ros_map_main.cc

```diff
@@ -26,16 +26,23 @@
 #include "cartographer/mapping_2d/probability_grid.h"
 #include "cartographer/mapping_2d/submaps.h"
 #include "cartographer/mapping_3d/submaps.h"
+#include "cartographer_ros/node_constants.h"
+#include "cartographer_ros/ros_log_sink.h"
 #include "cartographer_ros/ros_map.h"
 #include "cartographer_ros/submap.h"
 #include "gflags/gflags.h"
 #include "glog/logging.h"
+#include "ros/ros.h"
 
 DEFINE_string(pbstream_filename, "",
               "Filename of a pbstream to draw a map from.");
 DEFINE_string(map_filestem, "map", "Stem of the output files.");
 DEFINE_double(resolution, 0.05, "Resolution of a grid cell in the drawn map.");
 
+// Hacky way of providing online map from pbstream
+DEFINE_bool(publish_map, false, "Provide map topic instead of exporting");
```

Comment:
Expand description along the lines of "Publish map as ros topic "map" rather than generating pgm/yaml."